python/cuml/fil/fil.pyx

```diff
@@ -178,6 +178,7 @@ cdef extern from "cuml/fil/fil.h" namespace "ML::fil":
         bool output_class
         float threshold
         storage_type_t storage_type
+        int blocks_per_sm
```

Comment:
Just a suggestion, it would be nice to have a definition of `blocks_per_sm` in the cython file as well.